From 9608ef02519db7648cdede4f113e760e4540c0b9 Mon Sep 17 00:00:00 2001
From: Scott Little <scott.little@windriver.com>
Date: Mon, 2 Oct 2017 14:49:27 -0400
Subject: [PATCH 06/13] WRS:
 0001-Fix-Interfaces-intermittenly-do-not-come-up-on-boot.patch

Conflicts:
	SPECS/initscripts.spec
---
 SPECS/initscripts.spec | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/SPECS/initscripts.spec b/SPECS/initscripts.spec
index 5ba669b..63ad994 100644
--- a/SPECS/initscripts.spec
+++ b/SPECS/initscripts.spec
@@ -4,7 +4,7 @@ Version: 9.49.39
 # ppp-watch is GPLv2+, everything else is GPLv2
 License: GPLv2 and GPLv2+
 Group: System Environment/Base
-Release: 1%{?dist}.5
+Release: 1%{?dist}.6
 URL: https://github.com/fedora-sysv/initscripts
 Source: https://github.com/fedora-sysv/initscripts/archive/%{version}.tar.gz#/%{name}-%{version}.tar.gz
 
@@ -47,6 +47,7 @@ Patch4: support-interface-scriptlets.patch
 Patch5: relocate-dhclient-leases-to-var-run.patch
 Patch6: dhclient-restrict-interfaces-to-those-on-c.patch
 Patch7: support-interface-promisc.patch
+Patch8: 0001-dhclient-remove-1-arg.patch
 
 %description
 The initscripts package contains basic system scripts used
@@ -71,6 +72,7 @@ Currently, this consists of various memory checking code.
 %patch5 -p1
 %patch6 -p1
 %patch7 -p1
+%patch8 -p1
 
 %build
 make
-- 
1.9.1

